docs/how_it_works.rst

```diff
@@ -14,38 +39,57 @@ these as *passive* and *active* discovery.
 * Active discovery is invoked when a machine attempts to PXE boot on the network that RackHD is
   monitoring. As a new machine PXE boots, RackHD retrieves the MAC address of the machine.
   If the MAC address has not been recorded, RackHD creates a new record in the data model and
-  then invokes a default discovery workflow.
+  then invokes a default workflow. To enable active discovery, you set the default workflow that
+  will be run when a new machine is identified to one of the discovery workflows included
+  within the system. The most common is the SKU Discovery workflow.
 
-The discovery workflow is configured to download a pre-built Linux kernel, initrd and ubuntu filesystem
-to run in memory and coordinate with the monorail engine to run commands on the remote machine
-that interrogate its motherboard.
+For an example, the "SKU Discovery" workflow runs through its tasks as follows:
 
-Discovery Workflow
----------------------
+1. It runs a sub-workflow called 'Discovery'
 
-Discovery tasks are performed sequentially:
+   a) Discovery is initiated by sending down the iPXE boot loader with a pre-built script to run
+      within iPXE. This script then chainloads into a new, dynamically rendered iPXE script that interrogates
+      the enabled network interfaces on the remote machine and reports them back to RackHD. RackHD adds
+      this information to the machine and lookup records. RackHD then renders an additional iPXE script
+      to be chainloaded that downloads and runs the microkernel. The microkernel boots up and requests a
+      Node.js "bootstrap" script from RackHD. RackHD runs the bootstrap program which uses a simple REST
```

Comment:
Perhaps something like "requests a Node.js "bootstrap" script from RackHD, and runs the bootstrap program" since the node is running bootstrap.js, not RackHD?